modules/system/flake-overrides.nix

```diff
@@ -11,6 +11,9 @@ in
     system.checks.verifyNixPath = mkDefault false;
     system.checks.verifyNixChannels = mkDefault false;
 
+    # Still overridable with lib.mkForce
+    nix.nixPath = lib.mkDefault [ ];
```

Comment:
Won't this make specifying `nix.nixPath` mandatory for flake configurations?
I think wether a system uses `<nixpkgs>` as a channel or not is completely separate from if the configuration itself happens to be built using flakes.

Reply:
I'll remove this from this PR, the reason why I added it in the first place was primarily because `NIX_PATH` contained `darwin-configuration=` which was not relevant when using flakes. I think it would be good in future to have some more granular options for controlling the values added to `nix.nixPath`.

Reply:
Removed :+1:

Reply:
Ah, luckily nix.nixPath can also be an attribute set so I think it shouldn't be that hard to conditionally add the configuration path.